data/sql/updates/pending_db_world/update-sai-ssc-trash.sql

```diff
@@ -0,0 +1,120 @@
+(21964, 0, 4, 5, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Aggro - Set In Combat With Zone'),
+(21964, 0, 5, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 39, 10, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Aggro - Set In Combat With Zone'),
+(21964, 0, 6, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 38451, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Death - Cast Power of Caribdis'),-- N/A note: missing ability 38449 (on hp%?)
+(21965, 0, 0, 0, 0, 0, 100, 0, 10900, 10900, 10900, 14700, 0, 0, 11, 38234, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Frost Shock'),-- fully sniffed
+(21965, 0, 1, 0, 0, 0, 100, 0, 17000, 17000, 13350, 24250, 0, 0, 11, 38304, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Earthbind Totem'),-- fully sniffed
+(21965, 0, 2, 0, 0, 0, 100, 0, 15000, 15000, 30000, 30000, 0, 0, 11, 38306, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Poison Cleansing Totem'),-- not sniffed
+(21965, 0, 3, 0, 0, 0, 100, 0, 15800, 15800, 24250, 76750, 0, 0, 11, 38236, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Spitfire Totem'),-- fully sniffed
+(21965, 0, 4, 5, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - On Aggro - Set In Combat With Zone'),
+(21965, 0, 5, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 39, 10, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - On Aggro - Set In Combat With Zone'),
+(21965, 0, 6, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 38452, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - On Death - Cast Power of Tidalvess'),
+(21966, 0, 0, 0, 0, 0, 100, 0, 0, 1000, 2000, 2000, 0, 0, 11, 38374, 64, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - In Combat - Cast Hurl Trident'),-- not sniffed
+(21966, 0, 1, 0, 0, 0, 100, 0, 53400, 53400, 6050, 22250, 0, 0, 11, 29436, 0, 0, 0, 0, 0, 5, 40, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - In Combat - Cast Leeching Throw'),-- fully sniffed
+(21966, 0, 2, 0, 0, 0, 100, 0, 20650, 20650, 12150, 26350, 0, 0, 11, 38366, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - In Combat - Cast Multi-Toss'),-- fully sniffed
+(21966, 0, 3, 0, 0, 0, 100, 0, 14550, 14550, 60650, 84850, 0, 0, 11, 38431, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - In Combat - Cast Summon Fathom Sporebat'),-- fully sniffed -> now repeatable
+(21966, 0, 4, 0, 0, 0, 100, 0, 44900, 44900, 30300, 60500, 0, 0, 11, 38433, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - In Combat - Cast Summon Fathom Lurker'),-- fully sniffed -> now repeatable
```

Comment:
Not too sure about summons. Try checking Mangos first, see what they have. These high timers are telling me that they should actually respawn or start the timer on the death of the previous spawn.
Also these are part of a boss, no? I think they should be refactored into cpp in that case.